cypress/integration/devices_page_test.js

```diff
@@ -0,0 +1,54 @@
+describe('Devices page tests', () => {
+  context('no access before login', () => {
+    it('redirects to home', function () {
+      cy.visit('/devices');
+      cy.location('pathname').should('eq', '/login');
+    });
+  });
+
+  context('authenticated', () => {
+    beforeEach(() => {
+      cy.fixture('devices').as('devices');
```

Comment:
this represents a detailed device list, let's use a more specific name, for example `device_list_detailed` or `detailed_devices` to name a couple. Normally when queering for Astarte devices, you only obtain a list of IDs.
We can still save the fixture as `devices` though